mapit_gb/controls/2017-10.py

```diff
@@ -0,0 +1,59 @@
+# A control file for importing Boundary-Line.
+
+from mapit.models import Area, Generation
+
+
+def code_version():
+    return 'gss'
+
+
+def check(name, type, country, geometry, ons_code, commit, **args):
+    """Should return True if this area is NEW, False if we should match"""
```

Comment:
Could this comment be expanded to describe why it would return an Area instead of True/False?

Reply:
Ah yes, that comment is a bit out of date, the one in 2016-05.py is better!